api/insight/apirouter.go

```diff
@@ -50,6 +49,10 @@ func NewInsightApiRouter(app *InsightApi, useRealIP, compression bool) ApiMux {
 		mux.Use(middleware.NewCompressor(3).Handler())
 	}
 
+	mux.Get("/", func(w http.ResponseWriter, r *http.Request) {
+		http.Redirect(w, r, r.URL.Path+"/status", http.StatusSeeOther)
```

Comment:
Not redirecting to `"/insight/api/status"` because a server may have `"/insight"` mounted at the root.